shared/fs/header/header.desktop.js

```diff
@@ -106,27 +109,36 @@ const styleFolderHeaderEnd = {
   alignItems: 'center',
   paddingLeft: 16,
   paddingRight: 16,
+  flexShrink: 0,
 }
 
 const folderBreadcrumbStyle = {
   ...globalStyles.flexBoxRow,
   alignItems: 'center',
   paddingLeft: 0,
   paddingRight: 0,
+  flexShrink: 0,
 }
 
+const lastFolderBreadcrumbStyle = {
+  ...folderBreadcrumbStyle,
+  flexShrink: 1,
+}
+
+const stylesLastNameText = {}
```

Comment:
not sure how to fix this...maybe someone else with better CSS-fu can help? @buoyad perhaps?

Reply:
this should belong in a storybook
i'm not 100% what your trying to do here but can't you just do wordBreak: 'break-word'

Reply:
I had no luck with `overflowWrap`, didn't try `wordBreak` though

Reply:
I think overflowWrap and wordBreak are aliases to each other. Former is in the standard.

Reply:
@chrisnojima if you haven’t, please take a look at PR description. It worked in the pop up but not header.

Reply:
can you put this into a storybook then i'll look

Reply:
Alright; I added both header and row action popup to the storybook. Header is the one having problems. Thanks!!